server/db/models/Drink.js

```diff
@@ -0,0 +1,43 @@
+const {
+    Model,
+    DataTypes,
+  } = require('sequelize');
+  const db = require('../db');
+  
+class Drink extends Model {}
+
+Drink.init(
+{
+    id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true,
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      brand: {
+        type: DataTypes.STRING
+      }, 
+      type: {
+        type: DataTypes.STRING
+      },
+      price: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        }
```

Comment:
I think this needs `type: FLOAT` to store decimal values?

Reply:
yes, thank you